data/mships.txt

```diff
@@ -3948,8 +3948,8 @@ ship "CK80T Tlielart"
 	explode "large explosion" 40
 	explode "huge explosion" 10
 	"final explode" "final explosion large"
-	description `Massive freighter developed to carry as much cargo as possible, and did so exceeding the capacity of every other ship by far. Though unlike the other ship from MarStel Industries, it is very lightly armed relative to cargo capacity making it lacks the firepower to properly protect itself and neither did it have speed to dodge any raider. Due to such reason, it will require decent escort to safely travel.`
-	description `Though it carried more than most, if not all, freighter in human space the sheer amount of cargo that have to be unloaded will require ungodly amount of time, equipment, and manpower, which only get worse when loading cargo. Any less than well developed urban spaceport would struggle to service the ship and often will take a few days to complete the operation, if they ever require this much cargo in the first place. This results in the ship being not quite a popular choice in human space as few planets require this much cargo in a single delivery and fewer are capable of handling it.`
+	description `This is a massive freighter designed to carry as much cargo as possible, and does so, exceeding the capacity of every other ship by far. Though unlike the other ships from MarStel Industries, it is very lightly armed relative to cargo capacity, making it lack the firepower to properly protect itself, and it does not have the speed to dodge any raider. Due to such reasons, it will require decent escorts to safely travel.`
+	description `	Though it carries more than most, if not all, freighters in human space, the sheer amount of cargo that has to be unloaded requires ungodly amount of time, equipment, and manpower, which only gets worse when loading cargo. Any less than a well developed urban spaceport will struggle to service the ship and often will take a few days to complete the operation, if they ever require this much cargo in the first place. This results in the ship being not quite a popular choice in human space, as few planets require this much cargo in a single delivery and fewer still are capable of handling it.`
```

Comment:
```suggestion
	description `A massive freighter designed to carry as much cargo as possible, and did so exceedingly that it surpass the capacity of every other ships by far. Though unlike the other ships from MarStel Industries, it is very lightly armed relative to cargo capacity, making it lack the firepower to properly protect itself and neither does it have the speed to dodge any raider. Due to such reasons, it will require decent escort to safely travel.`
	description `	Though it carries more than most, if not all, freighters in human space, the sheer amount of cargo that has to be unloaded requires ungodly amount of time, equipment, and manpower, which only gets worse when loading cargo. Any less than a well developed urban spaceport will struggle to service the ship and often will take a few days to complete the operation, if they ever require this much cargo in the first place. This results in the ship being not quite a popular choice in most of human space, as few planets require this much cargo in a single delivery and fewer capable of handling it.`
```